.gitpod.yml

```diff
@@ -17,17 +18,28 @@ github:
     # add a label once the prebuild is ready to pull requests (defaults to false)
     addLabel: false
 tasks:
-  - init: pip3 install -e . && yarn install && yarn run build
-    # for some reason,have to re-install on start, or else it doesn't find the python package
+  - name: dev-mode
+    init: |
+      pip3 install -e ".[docs,test]"
+      jlpm install
+      jlpm run build
+      gp sync-done setup
     # Set no token and allow any origin, so that you can open it in a new tab
     # Disable iframe security so can load in the editor as well
-    command: >
-      pip3 install -e . &&
```

Comment:
Looks like this extra `pip install` can actually be removed, as it seems to be loading fine on startup:

![image](https://user-images.githubusercontent.com/591645/174675720-bf45c2e2-25d7-4965-97ba-8923a43b8a08.png)